src/tests/test_decorators.py

```diff
@@ -0,0 +1,39 @@
+from . import base
+from mock import patch
+import decorators
+
+class TestDecorators(base.BaseCase):
+    def test_timeit_smoke_test(self):
+        @decorators.timeit
+        def some_task():
+            pass
+        some_task()
```

Comment:
are we just aiming for code coverage? is there something here we can actually compare to an expected working state? perhaps have `some_task` sleep for a second and have the `timeit` decorator emit some value?